snap/snapcraft.yaml

```
@@ -0,0 +1,34 @@
+name: wsstat
+adopt-info: wsstat
+license: MIT
+summary: wsstat is a websocket traffic monitoring tool.
+description: |
+  wsstat provides a simple and easy to use tool to check the status and connection stats of a WebSocket endpoint.
+
+  Usage: wsstat wss://foo.example.com
+
+grade: stable
+confinement: strict
+base: core22
+
+apps:
+  wsstat:
+    command: bin/wsstat
+    plugs: [network, home]
+
+parts:
+  wsstat:
+    plugin: nil
+    build-snaps: 
+      - go
+    source: https://github.com/jakobilobi/wsstat.git
+    override-pull: |
+      craftctl default
+      craftctl set version="$(git describe --tags --abbrev=10)-$(git rev-parse --short HEAD)"
```

Apply suggestion to make
`wsstat_v0.1.1-1-g5759ee8a14-5759ee8_amd64.snap`
become
`wsstat_v0.1.1-1-g5759ee8-5759ee8_amd64.snap`
after building a snap on a non-tagged commit.

```suggestion
      craftctl set version="$(git describe --tags --abbrev=7)-$(git rev-parse --short HEAD)"
```